codecov.yml

```diff
@@ -18,10 +18,20 @@ coverage:
         paths:
           - OpenQA
           - backend/amt.pm
+          - backend/baseclass.pm
+          - backend/console_proxy.pm
+          - backend/driver.pm
+          - backend/generalhw.pm
+          - backend/ipmi.pm
           - backend/null.pm
+          - backend/qemu.pm
+          - backend/s390x.pm
           - backend/spvm.pm
+          - backend/svirt.pm
           - backend/vagrant.pm
           - backend/virt.pm
```

Comment:
How about using https://docs.codecov.com/docs/codecovyml-reference#ignore to only ignore pvm and ikvm and add `backend/` here?